src/gl.rs

```diff
@@ -433,6 +433,20 @@ pub fn get_integer_v(name: GLenum, data: &mut GLint) {
 }
 
 #[inline]
+pub fn get_boolean_v(name: GLenum, data: &mut GLboolean) {
+    unsafe {
+        ffi::GetBooleanv(name, data);
+    }
+}
+
+#[inline]
+pub fn get_float_v(name: GLenum, data: &mut GLfloat) {
+    unsafe {
+        ffi::GetFloatv(name, data);
```

Comment:
Just for consistency's sake, I'd make these functions return the GLvalue itself (just like `get_buffer_parameter_iv` et al.).

Apart from this I have no more complains, I hope @glennw can review it soon.